samza-yarn/src/main/scala/org/apache/samza/job/yarn/YarnJob.scala

```diff
@@ -88,25 +85,7 @@ class YarnJob(config: Config, hadoopConfig: Configuration) extends StreamJob {
   }
 
   def buildAmCmd() =  {
-    // figure out if we have framework is deployed into a separate location
-    val fwkPath = config.get(JobConfig.SAMZA_FWK_PATH, "")
-    var fwkVersion = config.get(JobConfig.SAMZA_FWK_VERSION)
-    if (fwkVersion == null || fwkVersion.isEmpty()) {
-      fwkVersion = "STABLE"
-    }
-    logger.info("Inside YarnJob: fwk_path is %s, ver is %s use it directly " format(fwkPath, fwkVersion))
-
-    var cmdExec = "./__package/bin/run-jc.sh" // default location
-
-    if (!fwkPath.isEmpty()) {
-      // if we have framework installed as a separate package - use it
-      cmdExec = fwkPath + "/" + fwkVersion + "/bin/run-jc.sh"
-
-      logger.info("Using FWK path: " + "export SAMZA_LOG_DIR=%s && ln -sfn %s logs && exec %s 1>logs/%s 2>logs/%s".
-             format(ApplicationConstants.LOG_DIR_EXPANSION_VAR, ApplicationConstants.LOG_DIR_EXPANSION_VAR, cmdExec,
-                    ApplicationConstants.STDOUT, ApplicationConstants.STDERR))
-
-    }
+    val cmdExec = "./__package/bin/run-jc.sh" // default location
```

Comment:
Minor: Maybe inline this method if not used in multiple places. If shared, can also just make this a class level val (constant).

Reply:
I noticed this too, but I will actually be adding more logic to this method in https://github.com/apache/samza/pull/1172, so I left it as is.